### PR TITLE
[MRG] FIX + API : on plot topo colors

### PR DESCRIPTION
**Pull request by agramfort**
FIX : fix colors on topo with black background
+
API : add dark_background parameter to show white background

### CODE REVIEW

File: mne/viz/evoked.py [hunk 1]
```diff
@@ -645,6 +664,42 @@ def plot_evoked_topo(evoked, layout=None, layout_scale=0.945, color=None,
...
+        warn('axis_facecolor is deprecated and will be removed in v0.16. Use '
+             'dark_background parameter or change matplotlib defaults.')
+    if axis_facecolor is None:
+        axis_facecolor = 'k'
+        warn('axis_facecolor is deprecated and will be removed in v0.16. Use '
+             'dark_background parameter or change matplotlib defaults.')
+    if font_color is None:
+        font_color = 'w'
+        warn('font_color is deprecated and will be removed in v0.16. Use '
+             'dark_background parameter or change matplotlib defaults.')
```
Comment (jaeilepp) [inline mne/viz/evoked.py]: These warnings get thrown when using the defaults. Shouldn't it be the other way round.

File: mne/viz/tests/test_topo.py [hunk 2]
```diff
@@ -120,8 +120,10 @@ def return_inds(d):  # to test function kwarg to zorder arg of evoked.plot
         _plot_update_evoked_topo_proj(params, bools)
     # should auto-generate layout
     plot_evoked_topo(picked_evoked_eeg.copy(),
-                     fig_background=np.zeros((4, 3, 3)), proj=True)
-    picked_evoked.plot_topo(merge_grads=True)  # Test RMS plot of grad pairs
+                     fig_background=np.zeros((4, 3, 3)), proj=True,
+                     dark_background=False)
+    # Test RMS plot of grad pairs
+    picked_evoked.plot_topo(merge_grads=True, dark_background=False)
```
Comment (jaeilepp) [inline mne/viz/tests/test_topo.py]: Looks like you're not testing ``dark_background=True`` case.

File: mne/viz/evoked.py [hunk 3]
```diff
@@ -637,6 +651,11 @@ def plot_evoked_topo(evoked, layout=None, layout_scale=0.945, color=None,
         See matplotlib documentation for more details.
     axes : instance of matplotlib Axes | None
         Axes to plot into. If None, axes will be created.
+    dark_background : bool
+        Plot on dark background (black) or not (white).
+        It will be set to False by default in v0.16
+
+        .. versionadded:: 0.15.0
```
Comment (Eric89GXL) [inline mne/viz/evoked.py]: I would rather have a generic `background_color` argument (or whatever we call it elsewhere). Then to decide the font color you just use the `np.mean(matplotlib.colors.colorConverter.to_rgb(background_color)) > 0.5` to decide whether to use black/white font

File: mne/viz/evoked.py (hunk 1 above)
Comment (agramfort) [inline mne/viz/evoked.py]: it's on purpose. Since it will be deprecated you need to set it manually before it breaks.

**Comment by agramfort** [comment]
done

File: mne/viz/evoked.py [hunk 4]
```diff
@@ -621,7 +622,7 @@ def plot_evoked_topo(evoked, layout=None, layout_scale=0.945, color=None,
         The figure face color. Defaults to black.
 
         .. note:: The parameter will be deprecated in version v0.16.
-                  Use dark_background parameter instead.
+                  Use background_color parameter instead.
```
Comment (Eric89GXL) [inline mne/viz/evoked.py]: will be removed, not will be deprecated

**Comment by agramfort** [comment]
good to go on my end

**Comment by agramfort** [comment]
fixed

**Comment by jaeilepp** [comment]
Thanks @agramfort

**Pull request merged**